packages/desktop-client/vite.config.mts

```diff
@@ -176,14 +178,19 @@ export default defineConfig(async ({ mode }) => {
           }),
       injectShims(),
       addWatchers(),
+      // react({
```

Comment:
I think we can remove this code instead of commenting it out. Apart from that LGTM